regress/core/tickets.sql

```diff
@@ -1114,6 +1114,16 @@ INSERT INTO bug_4144_table (geom)
 ANALYZE bug_4144_table;
 DROP TABLE IF EXISTS bug_4144_table;
 
+-- #4299
+SELECT '#4299', ST_Equals(ST_GeneratePoints(g, 1000), ST_GeneratePoints(g, 1000))
```

Comment:
what happens when you interleave seedless and seedful versions, can they rewrite seeds for one another?

Reply:
I'm now adding a few combinations of tests, all of these should yield true:
 * ST_Equals(ST_GeneratePoints(g, 1000, 12345), ST_GeneratePoints(g, 1000, 12345)),
 * ST_Disjoint(ST_GeneratePoints(g, 1000, 12345), ST_GeneratePoints(g, 1000, 54321)),
 * ST_Disjoint(ST_GeneratePoints(g, 1000, 12345), ST_GeneratePoints(g, 1000)),
 * ST_Disjoint(ST_GeneratePoints(g, 1000), ST_GeneratePoints(g, 1000))

Note that with the disjoint tests, if 1 or more points from either set of 1000 coincide, then that test fails. All seems good so far.